src/app/institutional/pages/institutional/institutional.component.scss

```diff
@@ -23,6 +22,10 @@ img {
   display: flex;
   align-items: center;
   justify-content: space-between;
+
+  .continue {
+    border: solid 1px rgba(0, 0, 0, 0.12);
```

Comment:
please use the colors on a separate theme .scss

Reply:
I was just reviewing the Register `next` button and doesn't have the border. I will remove it in the institutional so we will have both buttons with the same styles